tests/__init__.py

```diff
@@ -1,11 +1,24 @@
+from dataclasses import dataclass
 from typing import Optional
 
 import gym
 import numpy as np
 import phantom as ph
 
 
+class MockSampler(ph.utils.samplers.Sampler[float]):
+    def __init__(self, value: float) -> None:
+        self.value = value
+
+    def sample(self) -> float:
+        return self.value
```

Comment:
can you change that to `self.value + 1` for example to make the test more realistic please?

Reply:
I'm not sure how this improves the tests? The `MockSampler` is just an identity Sampler.

Reply:
it is to guarantee that the value is actually modified when the sampler.sample method is called, and that all the components are using the latest value as opposed to a spurious copy